src/gc.c

```diff
@@ -785,21 +785,21 @@ static inline int maybe_collect(void)
 
 // preserved values
 
-DLLEXPORT int jl_gc_n_preserved_values(void)
+int jl_gc_n_preserved_values(void)
```

Comment:
These were the ones I asked about originally, that were exported, but not used anywhere in Base or registered packages. I wasn't aware of the embedded document when I started... these functions aren't described there, do they need to be exported?